Cargo.toml

```diff
@@ -42,7 +36,7 @@ search = ["dep:regex"]
 arbitrary = { version = "1", features = ["derive"], optional = true }
 crossterm = { package = "crossterm", version = "0.27", optional = true }
 crossterm-025 = { package = "crossterm", version = "0.25", optional = true }
-ratatui = { version = ">=0.23.0, <1", default-features = false, optional = true }
+ratatui = { version = "0.24.0", default-features = false, optional = true }
```

Comment:
I feel `>=0.23.0, <1` is more correct. There is no reason to remove support for ratatui 0.23.

Reply:
Ah - got it. I misread the `>=`